api/src/main/java/org/apache/iceberg/view/ViewBuilder.java

```diff
@@ -18,18 +18,16 @@
  */
 package org.apache.iceberg.view;
 
-import java.util.List;
 import java.util.Map;
 import org.apache.iceberg.Schema;
-import org.apache.iceberg.catalog.Namespace;
 import org.apache.iceberg.catalog.ViewCatalog;
 
 /**
  * A builder used to create or replace a SQL {@link View}.
  *
  * <p>Call {@link ViewCatalog#buildView} to create a new builder.
  */
-public interface ViewBuilder {
+public interface ViewBuilder extends VersionBuilder<ViewBuilder> {
```

Comment:
Do we need a `withLocation` to set the location used for storing metadata files?

Reply:
At this point I'm not entirely sure whether we'd need to have a `withLocation()` method here. Should we wait until we get to https://github.com/apache/iceberg/pull/7913 and we can re-evaluate and add it when necessary?